src/lib/visualization/lqp_visualizer.cpp

```diff
@@ -27,9 +27,6 @@ namespace hyrise {
 LQPVisualizer::LQPVisualizer() {
   // Set defaults for this visualizer
   _default_vertex.shape = "rectangle";
-
-  // We can guarantee the LQP never changes during visualization and thus avoid redundant estimations for subplans
-  _cardinality_estimator.guarantee_bottom_up_construction();
```

Comment:
Note: We _can_ actually guarantee it again now that statistics pruning can be turned off and on.